Index: src/GuessClass.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/GuessClass.java	(date 1569229991000)
+++ src/GuessClass.java	(date 1569244856792)
@@ -8,7 +8,6 @@
     void guess(){
 
 
-
         while (numOfGuess<4) {
             System.out.println ("попытка номер "+numOfGuess );
             Scanner scanner= new Scanner (System.in);
@@ -18,25 +17,72 @@
             switch (s) {
 
                 case "lamp":
-                    System.out.println ("Верно !" + "");/// edfvvdvdv
+                    System.out.println ("Верно !" + "");
+                    System.out.println ("wanna game? Y/N" );
+                    String s2 = scanner.nextLine ();
+                    System.out.println (s2 );
+                    if (s2.equals ("y")){                      //if (s2=="y") not work
+                        System.out.println ("играем снова" );  //обнулить счетчики
+                        guess ();
+                    }
+                    else
+                        System.out.println ("game over" );
+
                     numOfGuess=4;
+
                     break;
 
                 case "i dont know":
                     System.out.println ("это лампа////");
+
+                    System.out.println ("wanna game? Y/N" );
+                    String s3 = scanner.nextLine ();
+                    if (s3.equals ("y")){
+                        System.out.println ("играем снова" );
+                        guess ();
+                    }
+                    else
+                        System.out.println ("game over" );
                     numOfGuess=4;
                     break;
 
                 default:
+                    System.out.println ("default running" );
                     numOfGuess++;
                     guessLeft--;
-                   if (guessLeft==2) {
-                       System.out.println ("не угадал/ попробуй снова," +" " + guessLeft+"попытки осталось");}
-                   if (guessLeft== 1){
-                       System.out.println ("не угадал/ попробуй снова," +" " + guessLeft+"попытка осталась");
-                   }
-
-                   if (guessLeft==0) System.out.println ("попытки закончились" );
+
+                    switch (guessLeft){
+
+                        case 2:
+                            System.out.println ("не угадал/ попробуй снова," +" " + guessLeft+"попытки осталось" );
+                            break;
+                        case 1:
+                            System.out.println ("не угадал/ попробуй снова," +" " + guessLeft+"попытка осталась");
+                            break;
+                        case 0:
+                             System.out.println ("попытки закончились" );
+
+                            System.out.println ("wanna game? Y/N" );
+                            String s4 = scanner.nextLine ();
+                            switch (s4){
+                                case "y":
+                                    numOfGuess= 1;
+                                    guessLeft = 3;
+                                    System.out.println ("starting game" );
+                                    guess ();
+                                    break;
+                                default:
+                                    System.out.println ("end game" );
+                            }
+                             break;
+                    }
+//                   if (guessLeft==2) {
+//                       System.out.println ("не угадал/ попробуй снова," +" " + guessLeft+"попытки осталось");}
+//                   if (guessLeft== 1){
+//                       System.out.println ("не угадал/ попробуй снова," +" " + guessLeft+"попытка осталась");
+//                   }
+//
+//                   if (guessLeft==0) System.out.println ("попытки закончились" );
 
 
 
Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Main.java	(date 1569229991000)
+++ src/Main.java	(date 1569241760142)
@@ -2,6 +2,7 @@
 
 public class Main {
     public static void main(String[] args) {
+        boolean  bo = false;
         System.out.println ("висит груша нельзя скущать что это отгадай 3 попытки" );
         GuessClass guessClass= new GuessClass ();
 
